apps/content/core/src/main/java/com/adobe/dx/content/marketo/models/MarketoCompFooter.java

```diff
@@ -0,0 +1,151 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.dx.content.marketo.models;
+
+import static com.adobe.cq.xf.ExperienceFragmentsConstants.PN_FRAGMENT_PATH;
+import static com.day.cq.commons.jcr.JcrConstants.JCR_CONTENT;
+import static org.apache.commons.lang.StringUtils.EMPTY;
+
+import com.adobe.dx.content.marketo.models.internal.MarketoConfBasicInfo;
+import com.adobe.dx.utils.service.CloudConfigReader;
+import com.day.cq.wcm.api.Page;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.AbstractResourceVisitor;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.OSGiService;
+import org.apache.sling.models.annotations.injectorspecific.ScriptVariable;
+import org.jetbrains.annotations.NotNull;
+
+@Model(adaptables = SlingHttpServletRequest.class)
+public class MarketoCompFooter {
```

Comment:
me nitpicking here, but could we call that MarketoFooter or MarketoComponentFooter?

Reply:
+1